huggingface/README.md

```diff
@@ -105,5 +106,17 @@ python hf-ort.py --hf_model {hf_model} --run_config {run_config} --process_count
 ## FAQ
 ### Problem with Azure Authentication
 If there's an Azure authentication issue, install Azure CLI [here](https://docs.microsoft.com/en-us/cli/azure/) and run `az login --use-device-code`
-### `RuntimeError: CUDA out of memory` error
-Change to smaller batchsize, use `--model_batchsize` parameter
\ No newline at end of file
+
+### In case of `RuntimeError: CUDA out of memory` error
+The issue is most likely caused by hitting a HW limitation on the target, this can be mitigated by using the following switches
+
+`--model_batchsize` <parameter> - Change to smaller batchsize
+  
+`--max_steps` <parameter> - Change the number of training steps
```

Comment:
Also max_steps should not affect CUDA out of memory error. Only mode_batchsize and process_count may affect. You can give it a try and see if it's the case. We changed max_steps so that the run will finish fast and we can see the result soon